src/test/java/teammates/test/cases/logic/FeedbackQuestionsLogicTest.java

```diff
@@ -568,6 +595,23 @@ public void testGetFeedbackQuestionsForStudents() {
         assertEquals(actualQuestions, expectedQuestions);
     }
     
+    public void testGetFeedbackQuestionsForStudent() {
```

Comment:
Yup it was updated on line 475, and a question containing an instructor as a giver in the custom feedback path was added to `typicalDataBundle.json` as well